pkg/registry/storage/csidriver/strategy.go

```diff
@@ -60,6 +60,9 @@ func (csiDriverStrategy) PrepareForCreate(ctx context.Context, obj runtime.Objec
 	if !utilfeature.DefaultFeatureGate.Enabled(features.CSIServiceAccountTokenSecrets) {
 		csiDriver.Spec.ServiceAccountTokenInSecrets = nil
 	}
+	if !utilfeature.DefaultFeatureGate.Enabled(features.VolumeLimitScaling) {
```

Comment:
What if the user set the value while the feature gate is disabled?

Reply:
Then we will drop the field (or set it to `nil`) as we are doing here. We are already doing that here.